raster/src/main/scala/geotrellis/raster/render/ColorMap.scala

```diff
@@ -326,3 +326,31 @@ class DoubleColorMap(breaksToColors: Map[Double, Int], val options: Options = Op
       .mkString(";")
   }
 }
+
+/** A color map where the breaks are monotonically increasing integer values starting at zero.
+ * Primarily used for capturing and persisting indexed color maps in GeoTIFFs. */
+class IndexedColorMap(indexedColors: Seq[Int]) extends IntColorMap(
+  indexedColors.zipWithIndex.map(p ⇒ p._2 -> p._1).toMap
+) {
+  override def toString = getClass.getSimpleName + "(" +
+    colors.map(c ⇒ f"0x$c%02x").mkString(", ") + ")"
+}
+
+object IndexedColorMap {
+  // As with GDAL, we need to pack each 16 bit color channel into 8 bits.
+  // GDAL just shifts and masks, effectively converting a 0-65535 range
+  // to a 0-255 one. These conversion approaches are similar to those used
+  // in the GDAL TIFF driver.
+  private def downsample(c: Int) = (c >> 8 & 0xFF).toShort
+  private def upsample(c: Int) = (c * 257).toShort
+
+  /** Creates an IndexColorMap from sequence of RGB short values. */
+  private[raster] def fromTiffPalette(tiffPalette: Seq[(Short, Short, Short)]) = new IndexedColorMap(
```

Comment:
not sure there's a need to make this private to [raster], is there a good reason to hide it?